torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10650,6 +10681,15 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
            dtypes=floating_and_complex_types(),
            sample_inputs_func=sample_inputs_linalg_slogdet,
            decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],),
+    OpInfo('linalg.vander',
```

Comment:
This looks good but if we don't want to add an additional test then we probably want to add "reference inputs" for vander that test it vs. the NumPy reference on a wider range of inputs. The reference inputs typically yield from the sample inputs and then have a list of additional cases. This will autogenerate a test where those inputs are passed to PyTorch's implementation and the NumPy reference implementation and the results compared.

Reply:
I don't know which other cases we could want to test this function on, given that we already have a test for non-contiguous samples and so on. I think that the set of samples (12) are fairly comprehensive.

Reply:
OK